### PR TITLE
Add Mayi project

### PR DESCRIPTION
**Pull request by yashrajgupta**

### CODE REVIEW

File: entry/src/main/resources/base/element/string.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    {
+      "name": "mainability_description",
+      "value": "Java_Phone_Empty Feature Ability"
+    },
+    {
+      "name": "mainability_HelloWorld",
+      "value": "Hello World"
+    },
+    {
+      "name": "Camera",
```
Comment (samuel-abhishek) [inline entry/src/main/resources/base/element/string.json]: Please remove the unused strings

File: entry/src/main/java/com/thanosfisherman/mayi/MainAbility.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import ohos.aafwk.content.Intent;
+import com.thanosfisherman.mayi.slice.MainAbilitySlice;
+
+public class MainAbility extends Ability {
+
+    @Override
+    public void onStart(Intent intent) {
+        super.onStart(intent);
+        super.setMainRoute(MainAbilitySlice.class.getName());
+
```
Comment (samuel-abhishek) [inline entry/src/main/java/com/thanosfisherman/mayi/MainAbility.java:29]: Remove unnecessary whitespaces to avoid lint issues

File: README.md [hunk 3]
```diff
@@ -1,88 +1,102 @@
...
-
-This library aims to reduce boilerplate code needed to request permissions at runtime by featuring a simple chainable API designed the way I want it.
-
-Screenshot
------------
+# MayI
+MayI is yet another library that simplifies the process of requesting permissions at runtime for Harmony devices.
 
+## Screenshot
 ![Demo screenshot](mayi_screenshot.gif "gif demo")
```
Comment (samuel-abhishek) [inline README.md:6]: Please capture the gif in english for better readability and understandability.

File: entry/src/main/java/com/thanosfisherman/mayi/MainAbility.java (hunk 2 above)
Comment (samuel-abhishek) [inline entry/src/main/java/com/thanosfisherman/mayi/MainAbility.java]: Please remove the white spaces to avoid lint issues.

**Review by samuel-abhishek — COMMENTED** [review]
Please do the mentioned changes.

File: entry/src/main/resources/base/element/string.json (hunk 1 above)
Comment (yashrajgupta) [reply entry/src/main/resources/base/element/string.json]: Done!

File: README.md (hunk 3 above)
Comment (yashrajgupta) [reply README.md:6]: Done

File: entry/src/main/java/com/thanosfisherman/mayi/MainAbility.java (hunk 2 above)
Comment (yashrajgupta) [reply entry/src/main/java/com/thanosfisherman/mayi/MainAbility.java]: Done

**Review by samuel-abhishek — APPROVED** [review]
LGTM

**Pull request merged**